src/Permissions/Backend/App.vala

```diff
@@ -29,7 +29,8 @@ public class Permissions.Backend.App : GLib.Object {
             id: id
         );
 
-        find_name ();
+        var appinfo = new GLib.DesktopAppInfo (id + ".desktop");
```

Comment:
Can you elaborate on what you mean by that?

Reply:
If you install org.foo.bar from .deb then it will be in `/usr/share/application/org.foo.bar.desktop`, the flatpak installed as user will be in `~/.local/share/flatpak/exports/share/application/org.foo.bar.desktop` and the flatpak installed as admin will be in `/var/lib/flatpak/exports/share/application/org.foo.bar.desktop` so there is no guarantee that GLib.DesktopAppInfo (id + ".desktop"); is taking the right one

Reply:
That's fair I guess